documentation/book/ref-topic-operator-usage-recommendation.adoc

```diff
@@ -5,11 +5,38 @@
 [id='topic-operator-usage-recommendations-{context}']
 = Topic Operator usage recommendations
 
-* Be consistent and always operate on `KafkaTopic` resources or always operate on topics directly. Avoid routinely using both methods for a given topic.
-* When creating a `KafkaTopic` resource:
-** Remember that the name cannot be changed later.
-** Choose a name for the `KafkaTopic` resource that reflects the name of the topic it describes.
-** Ideally the `KafkaTopic.metadata.name` should be the same as its `spec.topicName`. To do this, the topic name will have to be a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^].
-* When creating a topic:
-** Remember that the name cannot be changed later.
-** It is best to use a name that is a link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/identifiers.md[valid {KubernetesName} resource name^], otherwise the operator will have to modify the name when creating the corresponding `KafkaTopic`.
+When working with topics, be consistent and always operate on either `KafkaTopic` resources or topics directly. Avoid routinely switching between both methods for a given topic.
+
+Use topic names that reflect the nature of the topic, and remember that names cannot be changed later.
+
+If creating a name directly for a topic, use a name that is a valid {KubernetesName} resource name, otherwise the Topic Operator will need to modify the name when creating the corresponding `KafkaTopic`.
```

Comment:
Let's be clear that:

* we create topics, which have a name, rather than "creating topic names".
* the name is only modified on the Kube side.

```suggestion
If creating a topic in Kafka, use a name that is a valid {KubernetesName} resource name, otherwise the Topic Operator will need to create the corresponding `KafkaTopic` with a different name which obeys the {KubernetesName} rules.
```